docs/source/quickstart-docker.rst

```diff
@@ -16,6 +18,14 @@ Prerequisites
 You should have `Docker`_ installed on a Linux/Unix based system.
 
 
+
+.. note::
+    This ``jupyterhub/jupyterhub`` docker image is only an image for running
```

Comment:
This note is still duplicated from the above. Do you mean to move the note, or replace it? Because it shouldn't be in both places.

Reply:
> This note is still duplicated from the above. Do you mean to move the note, or replace it? Because it shouldn't be in both places.

I mean to remove it sir

Reply:
Is there a reason you moved the note about what the image is for to the 'prerequisites' section?

Reply:
Yes, it is added to the subsection "Prerequisites". I'm not sure I understand the reason for moving it there, since it isn't related to the prerequisites.

Reply:
> Yes, it is added to the subsection "Prerequisites". I'm not sure I understand the reason for moving it there, since it isn't related to the prerequisites.

you are right sir. it does make sense being under the prerequisites